ibr/ibr-sdk/src/Visualization.js

```diff
@@ -118,4 +137,4 @@ Object.assign( Visualization.prototype, {
 
 } );
 
-export {Visualization, swap32};
+export {Visualization, swap32, ONE_POINT};
```

Comment:
same with swap32, maybe combine it with constants in a util file or keep constants separate, up to you

Reply:
done

Reply:
no need to export ONE_POINT from this file anymore :)